docs/reference/operator_parameters.md

```diff
@@ -595,3 +595,34 @@ scalyr sidecar. In the CRD-based configuration they are grouped under the
 
 * **scalyr_memory_limit**
   Memory limit value for the Scalyr sidecar. The default is `500Mi`.
+
+## Connection pool configuration
+
+Parameters are grouped under the `connection_pool` top-level key and specify
+default configuration for connection pool, if a postgres manifest requests it
+but do not specify some of the parameters. All of them are optional with the
+operator being able to provide some reasonable defaults.
+
+* **connection_pool_number_of_instances**
+  How many instances of connection pool to create.
+
+* **connection_pool_schema**
+  Schema to create for credentials lookup function.
+
+* **connection_pool_user**
+  User to create for connection pool to be able to connect to a database.
+
+* **connection_pool_image**
+  Docker image to use for connection pool deployment.
+
+* **connection_pool_max_db_connections**
+  How many connections the pooler can max hold.
+
+* **connection_pool_mode**
+  Default pool mode, session or transaction.
+
+* **connection_pool_default_cpu_request**
+  **connection_pool_default_memory_reques**
+  **connection_pool_default_cpu_limit**
+  **connection_pool_default_memory_limit**
```

Comment:
No, but I don't think they need to - probably it makes sense to make those always equal just by default, since connection pooler resource requirements are usually rather stable.